Unable to build NW.js distribution (missing natives_blob.bin)

NWJS Version: nw44
Operating System: Ubuntu

I've been working on building NW.js for ARM and it's been [going well](https://github.com/nwjs/nw.js/issues/1151#issuecomment-586319704) except for the final step.  I've been using the [buildbot](http://buildbot-master.nwjs.io:8010/builders/nw44_sdk_linux64) commands as reference.  The issue is that the [`dist` build target](http://buildbot-master.nwjs.io:8010/builders/nw44_sdk_linux64/builds/34/steps/ninja_release_dist/logs/stdio) fails with the following:

```
ninja: Entering directory `out/nw'
[1/2] ACTION //content/nw:dist(//build/toolchain/linux:clang_arm)
FAILED: new_package.re
python ../../content/nw/tools/package_binaries.py -p /root/nwjs/src/out/nw -a arm -m sdk -i /root/nwjs/src/third_party/icu/common/icudtl.dat
Begin copy file
copy file end
Begin compress file
compress end
Working on /root/nwjs/src/out/nw
../../content/nw/tools/make-nw-headers.py
nw-headers generated
Creating packages...
File does not exist:  /root/nwjs/src/out/nw/natives_blob.bin
ninja: build stopped: subcommand failed.
```

As best I can tell, it's not actually possible to build `natives_blob.bin`, so I'm wondering how the buildbot manages to succeed here.  

The file is created by [`third_party/node-nw/deps/v8/BUILD.gn`](https://github.com/nwjs/node/blob/7dc9b6d47f9b0cd0461df160fca8e1633b45ff24/deps/v8/BUILD.gn#L878), but node is built using GYP, not GN, so the file is never _directly_ included.  (It appears to have portions _indirectly_ included using [`third_party/node-nw/tools/v8_gypfiles/v8.gyp`](https://github.com/nwjs/node/blob/b4e4788a5a3d297598bd8ef9c980c70adfc8457f/tools/v8_gypfiles/v8.gyp).)

Even if node were built with GN, it only appears to be possible to build a _blank_ `natives_blob.bin` file:

- The  [`natives_blob` action](https://github.com/nwjs/node/blob/7dc9b6d47f9b0cd0461df160fca8e1633b45ff24/deps/v8/BUILD.gn#L878) depends on the [`js2c_extras` action](https://github.com/nwjs/node/blob/7dc9b6d47f9b0cd0461df160fca8e1633b45ff24/deps/v8/BUILD.gn#L829).
- The `js2c_extras` action passes its sources (`v8_extra_library_files`) to `tools/js2c.py`.
- The `v8_extra_library_files` variable is [asserted to be empty](https://github.com/nwjs/node/blob/7dc9b6d47f9b0cd0461df160fca8e1633b45ff24/deps/v8/BUILD.gn#L264).

I ran the actions' python scripts directly and the result was a 1-byte `natives_blob.bin` file, but somehow the builtbot releases contain an actual file?

--- a/tools/package_binaries.py
+++ b/tools/package_binaries.py
@@ -30,7 +30,7 @@ args = parser.parse_args()
 # Init variables.
 binaries_location = None        # .../out/Release
 platform_name = None            # win/linux/osx
-arch = None                     # ia32/x64
+arch = None                     # ia32/x64/arm
 step = None                     # nw/chromedriver/symbol
 skip = None
 nw_ver = None                   # x.xx
@@ -153,7 +153,6 @@ def generate_target_nw(platform_name, arch, version):
                            'nw',
                            'icudtl.dat',
                            'locales',
-                           'natives_blob.bin',
                            'v8_context_snapshot.bin',
                            'lib/libnw.so',
                            'lib/libnode.so',
@@ -172,12 +171,13 @@ def generate_target_nw(platform_name, arch, version):
             target['input'] += ['nacl_helper', 'nacl_helper_bootstrap', 'pnacl']
             if arch == 'x64':
                 target['input'].append('nacl_irt_x86_64.nexe')
-            else:
+            elif arch == 'ia32':
                 target['input'].append('nacl_irt_x86_32.nexe')
+            else:
+                target['input'].append('nacl_irt_{}.nexe'.format(arch))
             
     elif platform_name == 'win':
         target['input'] = [
-                           'natives_blob.bin',
                            'v8_context_snapshot.bin',
                            'd3dcompiler_47.dll',
                            'libEGL.dll',
@@ -219,7 +219,6 @@ def generate_target_nw(platform_name, arch, version):
             target['input'].append('chromedriver')
             target['input'].append('libffmpeg.dylib')
             target['input'].append('minidump_stackwalk')
-            target['input'].append('natives_blob.bin')
             target['input'].append('v8_context_snapshot.bin')
     else:
         print 'Unsupported platform: ' + platform_name

